experiments/pages/lightningModalLauncher/lightningModalLauncher.xml

```diff
@@ -0,0 +1,202 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true" theme="Lightning">
+	<models>
+		<model id="Contact" limit="1" query="true" createrowifnonefound="false" datasource="salesforce" sobject="Contact">
+			<fields>
+				<field id="Id"/>
+				<field id="Name"/>
+			</fields>
+			<conditions/>
+			<actions/>
+		</model>
+		<model id="FlowParams" limit="20" query="true" datasource="Ui-Only" createrowifnonefound="true">
+			<fields>
+				<field id="name" displaytype="TEXT" length="255"/>
+				<field id="type" displaytype="TEXT" length="255"/>
+				<field id="value" displaytype="TEXT" length="255"/>
+			</fields>
+			<conditions/>
+			<actions/>
+		</model>
+		<model id="Flow" limit="20" query="true" datasource="salesforce" sobject="FlowInterview">
```

Comment:
I don't _think_ this model is used in the example, so I _think_ it can be deleted. If it is needed though, I'd make the sObject more generic and add a comment above it about what the model is for, how to adapt it to your Org.